_posts/2016-03-07-shaping-an-open-source-editorial-team.md

```diff
@@ -0,0 +1,74 @@
+
+Beginning this week, we’re going to start a weekly hangout on [Wednesdays at 9:30 am (EST)](http://everytimezone.com/#2016-3-9,-1290,cn3). The idea is to get the editorial team together to hangout and toss ideas around, get feedback, or just catch up/get to know each other. Feel free to join us! 
+
+## Where you can find us
+If you feel like joining an interesting community, and you have some writerly skills to contribute, but not only that, maybe you have social media skills, or organizational ones, or process writing, documentation writing, or event planning skills. The thing about Hoodie is, if you want to be a part of the community, there is a place for you. The only stipulation is, that you have to adhere to our [Code of Conduct](http://hood.ie/code-of-conduct).
+
+Ways you can find us –
++ [Check out the official contribute page](http://hood.ie/contribute/) 
++ [Visit our GitHub repository](https://github.com/hoodiehq/editorial)
++ [Sign up for Slack and come hang out to chat!](http://hood.ie/chat/)
+
+## What sorts of things has the team done so far?
+Only created the [most epic return of TGIF!](http://hood.ie/blog/terrier-open-source-sensitive-democracy-for-hamilton-tiles-tgif-73.html), Hoodie’s weekly blog of awesomeness! 
+
+Also, we said [Happy Birthday to Hoodie](http://hood.ie/blog/happy-birthday-hoodie.html) in February, as Hoodie turned three and we started gearing up for the Camp Release. 
+
+The editorial team also pushed forward the continuation of Meet the Hoodies blog series, a small spotlight on our regular contributors! Check out the one on [Charlotte Spencer](http://hood.ie/blog/Charlotte-Spencer-meet-the-hoodies-2.html), and [Alex Feyerke](http://hood.ie/blog/Alex-Feyerke-meet-the-hoodies-1.html)
+
+First things first, our editorial team had to have style. So we created a [style guide and wiki](https://github.com/hoodiehq/editorial/wiki) to be able to point to some general guidelines around writing and references. 
+
+## How you can contribute
+As mentioned above, there are several spots you can go to first, but if you’re excited to get started, skip all those and head to the [issues section in our repository](https://github.com/hoodiehq/editorial/issues).
+
+Examples of some open issues (or available writing contributions):
+
++ [Idea for blog series: projects we love](https://github.com/hoodiehq/editorial/issues/25)
++ [Creating our wikipedia page](https://github.com/hoodiehq/editorial/issues/57)
++ [Creating bios for our Hoodie animals](https://github.com/hoodiehq/editorial/issues/66)
++ [Help contribute to the hoodiehq twitter account](https://github.com/hoodiehq/editorial/issues/5)
++ [Write blog posts for the hoodie blog](https://github.com/hoodiehq/editorial/issues/71)
+
+Any of the issues with a `ready` label mean that they are available for anyone to take, so feel free to claim any of the issues you’re interested in.
```

Comment:
should we tell them about 'help wanted' instead of 'ready''? help wanted is more obvious in my opinion